### DISCUSSION TITLE
Automated fee rebalancing

### DISCUSSION THREAD

**Discussion by Benny-444** [Ride-The-Lightning/RTL#1380] [Ideas]
Does anybody know if there are plans to implement some type of automated fee management feature?

**Comment by saubyk** [comment]
Will happily accept prs in this area

But you have to keep certain things in mind:
- majority of the users, operating with a couple of channels on their node do not require automated fee management
- automated fee management is typically executed by profitable routing nodes operating at scale
- users who employ it will not give away the alpha for free (or as open source software). As it provides competitive advantage

that being said, there is an autofee solution available on Lightning Terminal Web, which you can use:
https://docs.lightning.engineering/lightning-network-tools/lightning-terminal/autofees

**Reply by Benny-444** [reply]
Thank you very much
